File: AdvantageFramework/Database/Entities/EmployeePicture.vb
Namespace Database.Entities

	<Table("EMPLOYEE_PICTURE")>
	Public Class EmployeePicture
		Inherits BaseClasses.Entity

#Region " Constants "



#End Region

#Region " Enum "

		Public Enum Properties
			ID
			EmployeeCode
			Image
			Nickname
			Wallpaper
			Employee

		End Enum

#End Region

#Region " Variables "



#End Region

#Region " Properties "

		<Key>
		<System.ComponentModel.DataAnnotations.Schema.DatabaseGeneratedAttribute(ComponentModel.DataAnnotations.Schema.DatabaseGeneratedOption.Identity)>
		<Required>
		<Column("EMP_PICTURE_ID")>
		<AdvantageFramework.BaseClasses.Attributes.Entity(IsRequired:=False, DisplayFormat:="")>
		Public Property ID() As Integer
		<Required>
		<MaxLength(6)>
		<Column("EMP_CODE", TypeName:="varchar")>
		<AdvantageFramework.BaseClasses.Attributes.Entity(IsRequired:=False, DisplayFormat:="")>
		Public Property EmployeeCode() As String
		<Column("EMP_IMAGE")>
		<AdvantageFramework.BaseClasses.Attributes.Entity(IsRequired:=False, DisplayFormat:="")>
		Public Property Image() As Byte()
		<MaxLength(10)>
		<Column("EMP_NICKNAME", TypeName:="varchar")>
		<AdvantageFramework.BaseClasses.Attributes.Entity(IsRequired:=False, DisplayFormat:="")>
		Public Property Nickname() As String
		<Column("EMP_WALLPAPER")>
		<AdvantageFramework.BaseClasses.Attributes.Entity(IsRequired:=False, DisplayFormat:="")>
		Public Property Wallpaper() As Byte()

        Public Overridable Property Employee As Database.Views.Employee

#End Region

#Region " Methods "

        Public Overrides Function ToString() As String

            ToString = Me.ID

        End Function

#End Region

	End Class

End Namespace

File: AdvantageFramework.Core/Database/Entities/EmployeePicture.cs

using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace AdvantageFramework.Core.Database.Entities
{
    [Table("EMPLOYEE_PICTURE")]
    public partial class EmployeePicture
    {
        #region Constants



        #endregion

        #region Enum

        public enum Properties
        {
            TBD
        }


        #endregion

        #region Variables



        #endregion

        #region Properties

        [Key]
        [Column("EMP_PICTURE_ID")]
        public int EmpPictureId { get; set; }
        [Required]
        [Column("EMP_CODE")]
        [StringLength(6)]
        public string EmpCode { get; set; }
        [Column("EMP_IMAGE", TypeName = "image")]
        public byte[] EmpImage { get; set; }
        [Column("EMP_NICKNAME")]
        [StringLength(10)]
        public string EmpNickname { get; set; }
        [Column("EMP_WALLPAPER", TypeName = "image")]
        public byte[] EmpWallpaper { get; set; }

        #endregion

        #region Methods



        #endregion
    }
}